OCPBUGS-908 updating custom seccomp profile procedures (#55225)
openshift/openshift-docs: security/seccomp-profiles.adoc

== Configuring a custom seccomp profile
You can configure a custom seccomp profile, which allows you to update the filters based on the application requirements. This allows cluster administrators to have greater control over the security of workloads running in OpenShift Container Platform.

> Seccomp security profiles list the system calls (syscalls) a process can make. Permissions are broader than SELinux, enabling users to restrict operations system-wide, such as `write`.

Reviewer: ```suggestion
Seccomp security profiles list the system calls (syscalls) a process can make. Permissions are broader than SELinux, which restrict operations, such as `write`, system-wide.
```
I might be misunderstanding the sentence, but I'm pretty sure this is conveying the same meaning
